src/asynk.rs

```diff
@@ -92,12 +92,15 @@ use std::fmt;
 use std::io;
 use std::net::IpAddr;
 use std::path::Path;
+use std::sync::atomic::Ordering;
 use std::sync::{atomic::AtomicBool, Arc};
 use std::time::Duration;
 
 use blocking::unblock;
 use crossbeam_channel::{Receiver, Sender};
 
+use crate::message::IntoBound;
```

Comment:
Hmm wonder if we need this `IntoBound` trait at all? Why would an end-user do this conversion?